lib/views/profiles_screen/profiles_screen.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+
+class ProfilesScreen extends StatelessWidget {
+  const ProfilesScreen({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Proflies View'),
+      ),
+      body: const Stack(
+        children: [
+          // Background image put the file here
+          DecoratedBox(
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage('assets/thc_logo.png'),
+                fit: BoxFit.cover,
+              ),
```

Comment:
Using the logo as a backdrop is an interesting design choice; we may swap for something else in the future.